src/components/dom.jsx

```diff
@@ -59,6 +60,10 @@ const DOM = (props) => {
 		escapedState,
 	};
 
+	if(clientFilename){
```

Comment:
I'm interested in the motivation for making this optional - in what case do you want to render the app without a client script?